Source/Shared/Swizzler.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+struct Swizzler {
+
+  enum Kind {
+    case Instance
+    case Class
+  }
+
+  static func swizzleMethod(method: String, cls: AnyClass!, prefix: String = "swizzled", kind: Kind = .Instance) {
```

Comment:
Could we rename it to `swizzle`? That you are swizzling a method is a part of the signature in the labels?